packages/core/src/Utils/Language/language.js

```diff
@@ -27,9 +33,5 @@ export const changeLanguage = (key, changeCurrentLanguage) => {
             new_url.searchParams.set('lang', key);
         }
         window.history.pushState({ path: new_url.toString() }, '', new_url.toString());
-        changeLanguageTranslation(key, () => {
-            changeCurrentLanguage(key);
-            BinarySocket.closeAndOpenNewConnection(key);
```

Comment:
@aizad-deriv, If you left the call to `BinarySocket.closeAndOpenNewConnection(key);` here, would your solution still work?